### ISSUE TITLE
REST Api: Relationship information for linked releases of a release

### ISSUE THREAD

**Issue by Lythenas** [eclipse/sw360#1225]
[//]: # (Copyright Siemens AG, 2021. Part of the SW360 Portal Project)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

### Description

I want to fetch all transitive linked releases of project with all the associated relationship data (comment and relation, etc). Currently this is impossible.

With `/api/projects/376576/releases?transitive=true` the relationship information is not present. The data is only in the `linkedReleases` field when you fetch the whole project. This works fine for direct linked releases. But this does not work for transitive linked releases.

I'm trying to manually walk the transitive linked releases. But the component releases (`/api/releases/<id>`) does not return a `linkedReleases` field like project does. It only has an embedded `sw360:releases` which is missing the relationship information.

For reference the linkedReleases field on projects:

```
  "linkedReleases" : [ {
    "createdBy" : "admin@sw360.org",
    "release" : "https://sw360.org/api/releases/3765276512",
    "mainlineState" : "MAINLINE",
    "comment" : "Test Comment",
    "createdOn" : "2020-08-05",
    "relation" : "CONTAINED"
  } ],
```

---

Versions

Last commit id on master: 13.3.0-M1
Operating System (lsb_release -a): Ubuntu 18.04

**Comment by KoukiHama** [comment]
Maybe #863 is related this issue.

**Comment by mcjaeger** [comment]
yes, linked released are not followed on project level currently, should be done.
